var/spack/repos/builtin/packages/discotec/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Discotec(CMakePackage):
+    """This project contains DisCoTec, a code for the distributed sparse
+    grid combination technique with MPI parallelization."""
+
+    homepage = "https://github.com/SGpp/DisCoTec"
+    url = "https://github.com/SGpp/DisCoTec"
+    git = "https://github.com/SGpp/DisCoTec"
+
+    # notify when the package is updated.
+    maintainers = ["freifrauvonbleifrei", "pfluegdk"]
+
+    version("main", branch="main")
+
+    variant("ft", default=False, description="DisCoTec with algorithm-based fault tolerance")
+    variant("gene", default=False, description="Build for GENE (as task library)")
+    variant("hdf5", default=True, description="Interpolation output with HDF5")
+    variant("lto", default=True, description="Build with link-time optimization")
+    variant("openmp", default=True, description="Parallelize with OpenMP")
+    variant("timing", default=True, description="With high-res timers")
+    variant("test", default=True, description="Build Boost tests")
+    variant("selalib", default=False, description="Build selalib example")
+    variant("vtk", default=False, description="Build with VTK support")
+
+    depends_on(
+        "boost +test +serialization +filesystem +system +program_options", type=("build", "run")
```

Comment:
Same question here for the `build,run` type.

Reply:
hm, I assume the flags could be removed. let me try ;)